Panic when cloning pwstore

### Summary
When cloning a password store on a fresh install using `gopass clone` with gopass v1.9.2 on Ubuntu 18.04 gopass throws a segfault.
```
➜  ~ gopass clone git@github.com:<repo>
Configuring git repository ...
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x88 pc=0x838bd7]

goroutine 1 [running]:
github.com/gopasspw/gopass/internal/cui.AskForGitConfigUser(0xd5b200, 0xc0002122d0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x1, 0x28, 0x0)
	/home/gnosthi/go/src/github.com/gopasspw/gopass/internal/cui/recipients.go:243 +0x37
github.com/gopasspw/gopass/internal/action.(*Action).cloneGetGitConfig(0xc000218000, 0xd5b200, 0xc0002122d0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
	/home/redacted/go/src/github.com/gopasspw/gopass/internal/action/clone.go:134 +0xaa
github.com/gopasspw/gopass/internal/action.(*Action).clone(0xc000218000, 0xd5b200, 0xc0002122d0, 0x7fffc693c429, 0x26, 0x0, 0x0, 0xc00020e1e0, 0x2d, 0xc000210601, ...)
	/home/redacted/go/src/github.com/gopasspw/gopass/internal/action/clone.go:93 +0x605
github.com/gopasspw/gopass/internal/action.(*Action).Clone(0xc000218000, 0xc000082580, 0x0, 0xc00020a1b0)
	/home/redacted/go/src/github.com/gopasspw/gopass/internal/action/clone.go:43 +0x2bf
github.com/urfave/cli.(*Command).Run(0xc00021e6c0, 0xc000082400, 0x0, 0x0)
	/home/redacted/go/src/github.com/urfave/cli/command.go:163 +0x4b9
github.com/urfave/cli.(*App).RunContext(0xc0001f0480, 0xd5b200, 0xc0002122d0, 0xc000098150, 0x3, 0x3, 0x0, 0x0)
	/home/redacted/go/src/github.com/urfave/cli/app.go:306 +0x790
main.main()
	/home/redacted/go/src/github.com/gopasspw/gopass/main.go:81 +0x3ae
```

### Steps To Reproduce
Starting from fresh install from src, setup git, (with commit signing), and clone a remote password store.

### Expected behavior
Password store should be cloned and setup

### Environment
<!--
Please complete the following information (see note below)
-->

- OS: Ubuntu 18.04
- OS version:  5.3.0-53-generic #47~18.04.1-Ubuntu SMP
- gopass Version: 1.9.2-git+HEAD
- Installation method: gopass repo (from source)

--- a/internal/cui/recipients.go
+++ b/internal/cui/recipients.go
@@ -240,6 +240,9 @@ func AskForPrivateKey(ctx context.Context, crypto backend.Crypto, prompt string)
 func AskForGitConfigUser(ctx context.Context, crypto backend.Crypto) (string, string, error) {
 	var useCurrent bool
 
+	if crypto == nil {
+		return "", "", fmt.Errorf("crypto not available")
+	}
 	keyList, err := crypto.ListIdentities(ctx)
 	if err != nil {
 		return "", "", err

